static/src/components/RegisterView.js

```diff
@@ -108,8 +107,13 @@ export default class RegisterView extends React.Component {
     }
 
     login(e) {
+        console.log(this.state.profile);
```

Comment:
Can we try and keep console.logs out of master now

Reply:
Yep. saw your last post. all good